convert2rhel/backup.py

```diff
@@ -374,6 +390,21 @@ def enable(self):
         # Set the enabled value
         super(NewRestorableFile, self).enable()
 
+    def _hash_backup_path(self):
```

Comment:
Maybe I would change a bit the name of the method or add to comment some example to help with understanding. Right now it feels to me, it will return the hash of the backup path. I would expect it will return the `hashed_directory = os.path.join(BACKUP_DIR, hashlib.md5(path.encode()).hexdigest())`

Reply:
I like the idea of adding an example of the usage. I will update it. 

Regarding the name, I will try to come up with something more explicit.